src/main/java/models/AccountModel.java

```diff
@@ -0,0 +1,32 @@
+package models;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Random;
+
+@Getter
+@Setter
+public class AccountModel {
+    public CustomerModel customerModel;
+    public Boolean isSubscribedToNewsletter;
+    public String loginName;
+    public String password;
+
+    public AccountModel(CustomerModel customerModel, Boolean isSubscribedToNewsletter) {
+        this.customerModel = customerModel;
+        this.isSubscribedToNewsletter = isSubscribedToNewsletter;
+        this.loginName = customerModel.getLogin();
+        this.password = customerModel.getPassword();
+    }
+
+    public AccountModel() {
```

Comment:
Why no builder here? Is is shouting - use me use me :)

Reply:
Something like that?
```java
public class AccountModel {
    public CustomerModel customerModel;
    public String loginName;
    public String password;

    @Builder.Default
    public Boolean isSubscribedToNewsletter = false;

    public static AccountModel withCustomerModel(CustomerModel customerModel) {
        return AccountModel.builder()
                .customerModel(customerModel)
                .loginName(CustomerModelUtils.generateLoginName())
                .password(CustomerModelUtils.generatePassword())
                .build();
    }

    public static AccountModel withRandomCustomerModel() {
        return AccountModel.builder()
                .customerModel(CustomerModelUtils.generateCustomerWithRandomData())
                .loginName(CustomerModelUtils.generateLoginName())
                .password(CustomerModelUtils.generatePassword())
                .build();
    }
}
```